docs/02-Install.md

```diff
@@ -153,11 +154,11 @@ Data loaded in this module gets populated automatically by the insertion
 scripts. As such, scans whose parameters can't be matched against the 
 `mri_protocol` table during the imaging insertion process, will be flagged as 
 protocol violations and will not have their MINC/NIfTI volumes loaded in the 
-database. The type of error (scan identification, protocol violation) will be 
-listed and can be reviewed from the front-end.
+Imaging Browser module. The type of error (scan identification, protocol 
+violation) will be listed and can be reviewed from the front-end.
```

Comment:
Modify to include: "Violated scans can be viewed and the type of error (scan identification, protocol violation) can be reviewed from the front-end."

Reply:
done